src/common/transformations/include/transformations/mark_dequantization_subgraph.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/pass/graph_rewrite.hpp>
```

Comment:
Use `""` instead `<>` as in other newly added files?
Can `openvino` includes be used?